examples/image_classifier/streaming_video/README.md

```diff
@@ -0,0 +1,147 @@
+# Streaming Video Inference
```

Comment:
Let's use a different folder for this, `examples/streaming_video` would be better

Reply:
Hmmm.. All the examples we have are classified according to the class of the AI algorithm being used...ex: image classifier, object detector etc and I think that makes sense..

Reply:
How about linking to it from examples/README.md just want to make sure it's discoverable

Reply:
Thats a good idea

Reply:
Done